pkg/api/types.go

```diff
@@ -221,6 +224,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDiskVolumeSource struct {
+	// Required: iSCSI target IP
+	TargetIP string `json:"targetIP,omitempty" description:"iscsi target IP address"`
```

Comment:
I believe descriptions are only for fields in the v1beta* packages.  Internal types do not include descriptions.